files/manifests/k8s-controller-manager.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     {{ else -}}
     - --cloud-provider={{.Cluster.Kubernetes.CloudProvider}}
     {{ end -}}
-    - --terminated-pod-gc-threshold=10
+    - --terminated-pod-gc-threshold={{if eq .ControllerManagerTerminatedPodGcThreshold 0}}125{{else}}{{.ControllerManagerTerminatedPodGcThreshold}}{{end}}
```

Comment:
`{{ default 125 .ControllerManagerTerminatedPodGcThreshold }}` should work as well, I guess